src/AdvApi32.Desktop/AdvApi32+SC_STATUS_TYPE.cs

```diff
@@ -9,7 +9,7 @@ namespace PInvoke
     public partial class AdvApi32
     {
         /// <summary>
-        /// Specifies the information level for the <see cref="QueryServiceStatusEx(SafeServiceHandle, SC_STATUS_TYPE, void*, int, out int)"/> method.
+        /// Specifies the information level for the <see cref="QueryServiceStatusEx(SafeServiceHandle, SC_STATUS_TYPE, System.IntPtr, int, out int)"/> method.
```

Comment:
I'm curious why you changed this. We generally prefer referencing the original overload rather than one of the code-generated ones. We're not picky there, but since you're deliberately changing it in the other direction, I'm curious what the motivation is.